cloudanalysis.fd/NonVarCldLib/cloud_saturation.f90

```diff
@@ -333,3 +342,72 @@ function ruc_saturation(Temp,pressure)
         ruc_saturation= (watwgt*qvs1 + (one-watwgt)*qvi1)  !  kg/kg
 !
 end function ruc_saturation
+
+!+---+-----------------------------------------------------------------+
+!>\ingroup aathompson
```

Comment:
This is now consistent with Thompson microphysics, but the doxygen comments should be modified.

Reply:
modified